includes/tracks/events/class-wc-admin-setup-wizard-tracking.php

```diff
@@ -20,43 +22,40 @@ class WC_Admin_Setup_Wizard_Tracking {
 
 	/**
 	 * Init tracking.
+	 *
+	 * @deprecated 4.6.0
 	 */
 	public function init() {
-		if ( empty( $_GET['page'] ) || 'wc-setup' !== $_GET['page'] ) { // WPCS: CSRF ok, input var ok.
-			return;
-		}
-
-		add_filter( 'woocommerce_setup_wizard_steps', array( $this, 'set_obw_steps' ) );
-		add_action( 'shutdown', array( $this, 'track_skip_step' ), 1 );
-		add_action( 'add_option_woocommerce_allow_tracking', array( $this, 'track_start' ), 10, 2 );
-		add_action( 'admin_init', array( $this, 'track_ready_next_steps' ), 1 );
-		add_action( 'wp_print_scripts', array( $this, 'dequeue_non_allowed_scripts' ) );
-		$this->add_step_save_events();
-		add_action( 'woocommerce_setup_footer', array( $this, 'add_footer_scripts' ) );
+		_deprecated_function( __CLASS__ . '::' . __FUNCTION__, '4.6.0', __( 'Onboarding is maintained in WooCommerce Admin.', 'woocommerce' ) );
```

Comment:
@rrennick one thing that I didn't notice in the initial review is that when tracking is enabled, `WC_Site_Tracking::init()` will call this method and a deprecated function warning will be displayed. Maybe we need to remove `WC_Admin_Setup_Wizard_Tracking` from the list of tracking classes?

https://github.com/woocommerce/woocommerce/blob/f2be9c7365911e34498ea9eb6157f4e61691db23/includes/tracks/class-wc-site-tracking.php#L166

Reply:
Removed.